cmds/webboot/utils.go

```diff
@@ -66,3 +71,28 @@ func download(URL, fPath string) error {
 	verbose("%q is downloaded at %q\n", URL, fPath)
 	return nil
 }
+
+// getCachedDirectory recognizes the usb stick that contains the cached directory from block devices,
+// and return it's mount point.
+func getCachedDirectory() (*mount.MountPoint, error) {
```

Comment:
This is a personal preference but this seems like something pretty specific to the weboot program and not necessarily a util, so I'd put it in webboot.go